domainlab/utils/generate_benchmark_plots.py

```diff
@@ -167,8 +170,8 @@ def scatterplot_matrix(dataframe_in, use_param_index, file=None, kind='reg',
     distinguish_param_setups: if True the plot will not only distinguish between models,
         but also between the parameter setups
     '''
-    dataframe = dataframe_in.iloc[:, G_DF_PLOT_COL_START:].copy()
-    index = list(range(5, dataframe.shape[1]))
+    dataframe = dataframe_in.copy()
+    index = list(range(G_DF_PLOT_COL_METRIC_START, dataframe.shape[1]))
```

Comment:
the original code selected "seed, params, acc, precision, recall, specificity, f1, auroc", since range(5, dataframe.shape[1]),
now the code use G_DF_PLOT_COL_METRIC_START=7 we do not have "seed" and "params" anymore, is this the expected behavior?

Reply:
Index is expected to collect the indices of the columns containing the objectives. 

In the previous version this were the columns with index 5 or higher as I cropped the first two columns at the beginning of the function. 
To make the code more easy to understand I now removed the cropping in the beginning of the function, hence now we can use the macro for METRIC_START here.